compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangParserListener.java

```diff
@@ -639,12 +639,12 @@ public void exitArrayTypeNameLabel(BallerinaParser.ArrayTypeNameLabelContext ctx
                 if (children.get(index + 1).getText().equals("]")) {
                     sizes.add(UNSEALED_ARRAY_INDICATOR);
                     index += 2;
-                } else if (children.get(index + 1) instanceof BallerinaParser.SealedLiteralContext) {
+                } else if (children.get(index + 1).getText().equals("*")) {
```

Comment:
Better we can use const value here, instead of a string.

Reply:
Added a const value with https://github.com/ballerina-platform/ballerina-lang/pull/14163/commits/b58dcf2630d4aacb53767112899c20e0e564323c.